smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftSettings.kt

```diff
@@ -64,6 +65,7 @@ class SwiftSettings(
     val mergeModels: Boolean,
     val copyrightNotice: String,
     val visibility: String,
+    val internalClient: Boolean,
```

Comment:
Added this `internalClient` field to SwiftSettings to help make codegen more clear, i.e. can query this setting directly rather than rely on some string value of `visibility`.